src/components/component-header/ComponentHeader.tsx

```diff
@@ -21,7 +23,7 @@ export const ComponentHeader: React.FC<Props> = (props: Props) => {
     <div className="component-header">
       <GoABadge type="information" content={props.category} />
       <h1>{props.name}</h1>
-      <h3>{props.description}</h3>
+      <h3><Markup content={props.description} /></h3>
```

Comment:
Instead of adding an external library for this, let's try the following first and if required create a component for it.
https://legacy.reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml